### ISSUE TITLE
If you press the space bar to write a commit message in Korean, the commit message is no longer written.

### ISSUE THREAD

**Issue by yourichng** [sublimehq/sublime_merge#772]
**Version info**

 - OS: Windows 10 2004 (OS Build 19041.264)
 - Build: Stable Channel, Build 2020

**Description**

A description of what the bug is.

**Steps to reproduce**

1. Write the commit message in Korean.
2. Press the space bar.
3. Continue writing the next message.

**Expected behavior**

I was using sublime merge well before.

However, after updating this time, if you use a space to write a commit message, the text is no longer entered.

English is not a problem, but Korean is still experiencing problems.

In the attached screenshot, I wrote "테스트 커밋", but after pressing "테스트" I couldn't enter anything anymore when I pressed the spacebar.

**Screenshots**

![ko](https://user-images.githubusercontent.com/2605918/83475335-18d1b600-a4c9-11ea-9938-c77210e63990.png)

**Comment by dpjohnst** [comment]
Hi @yourichng,

Thank you for reporting this issue.

This is due to a bug with the Korean IME - see: https://github.com/mozilla/gecko-dev/blob/baf1cd492406a9ac31d9ccb7a51c924c7fbb151f/widget/windows/IMMHandler.cpp#L722

We'll be adding a work-around for this issue in the meantime, and will keep you updated.

Thanks,
\- Dylan

**Comment by dpjohnst** [comment]
Hi @yourichng,

This issue has been fixed in development build 2024, and will be fixed in the next stable build.

Kind regards,
\- Dylan

**Comment by yourichng** [comment]
Hello @dpjohnst 

I updated the sublime merge to 2025 today, and confirmed that the Windows IME error was fixed in the history.

However, more serious problems were encountered than before.

Only the first Hangul character is entered, but none are entered.

For example, if you enter "test", only "t" is entered, followed by "est".

In Korean, if you enter "테스트.", "테." is being entered.

One word becomes more serious from what was put on, so only one letter can be entered.
